Jenkinsfile_k8s

```diff
@@ -38,15 +43,21 @@ if (env.BRANCH_IS_PRIMARY) {
     // Do not build concurently on the principal branch (to avoid Azure ARM issues with shared resources)
     disableConcurrentBuilds(),
   ])
+  currentVersionMajor = 1
 }
 
 if (env.CHANGE_ID) {
   properties([
     // Do not build concurently on pull requests (to avoid Azure ARM issues with shared resources), and abort previous running build
     disableConcurrentBuilds(abortPrevious: true)
   ])
+  currentVersionMinor = env.CHANGE_ID
 }
 
+def currentVersion = currentVersionMajor + '.' + currentVersionMinor + '.' + currentVersionPatch
+
+echo "current version: $currentVersion"
```

Comment:
In its current state, we are checking if using a suffix in the Azure gallery version works (to ensure a unique version per build).

If it works, then you should move this "logic" to a shell command right before the `packer` commands instead of pipeline groovy code: pipeline should only be glue code.

Reply:
I've put that in a similar format as the other PKR_VAR_